### ISSUE TITLE
add sundown info to php-doc.

### ISSUE THREAD

**Issue by chobie** [chobie/php-sundown#8]
it very useful for everyone

**Comment by stealth35** [comment]
https://wiki.php.net/doc/howto/pecldocs

:v:

**Comment by chobie** [comment]
yea, but no one publish with namespaced class yet AFAIK. o.O

**Comment by stealth35** [comment]
Yeah good question, HTTP has namespaces for the version 2, but nothing in the doc

**Comment by chobie** [comment]
functional-php guy tried to put docs. but there was no manual on docs.php.net.

http://old.nabble.com/DocGen%2C-PHPDOC%2C-PhD-and-namespaced-extensions.-tt31841428.html#a31841428

hm, I'll ask someone to create pecldocs later.
